updating/updating-cluster-between-minor.adoc

```diff
@@ -18,6 +18,21 @@ Because of the difficulty of changing update channels by using `oc`, use the web
 See xref:../authentication/using-rbac.adoc[Using RBAC to define and apply permissions].
 * Have a recent xref:../backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd backup] in case your upgrade fails and you must xref:../backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[restore your cluster to a previous state].
 
+// The following note will need to stay until (and be updated for) the 4.4 docs
+
+// Assuming next z-streams:
+// from {product-title} 4.1.31 or earlier; as of {product-title} 4.1.34
+// from {product-title} 4.2.16 or earlier; as of {product-title} 4.2.18
+
+[IMPORTANT]
+====
+If you are upgrading to this release from {product-title} 4.2.16 or earlier, you must restart all Pods after the upgrade is complete.
+
```

Comment:
As Eric suggested,we better show the step of restarting pods,like:
```
$ for I in $(oc get ns -o jsonpath='{range .items[*]} {.metadata.name}{"\n"} {end}'); \
      do oc delete pods --all -n $I; \
      sleep 1; \
      done
```

Reply:
@sferich888 So like the other place, I can add this step right here in the callout, but do you think it would be better placed to actually make it a step at the end of the update procedure, (like here [1]). Or do you think that people will catch this note at the top with the step?

[1] https://osdocs-878-auto-rotate--ocpdocs.netlify.com/openshift-enterprise/latest/updating/updating-cluster-between-minor.html#update-upgrading-web_updating-cluster-between-minor